pages/common/sha256sum.md

```diff
@@ -22,3 +22,7 @@
 - Only show a message for files for which verification fails:
 
 `sha256sum --check --quiet {{path/to/file.sha256}}`
+
+- Only show a message for files that are there and for which verification fails:
```

Comment:
I think this is more clear for non-native English speakers
```suggestion
- Only show a message for files for which verification fails ignoring missing files:
```

Reply:
Well, I'm a non-native English speaker ;) And I agree with you. I didn't like my initial wording _"that are there"_ but struggled to come up with a better phrase. I think your edit is better but I want to add a comma after _"fails"_.

**Only show a message for files for which verification fails, ignoring missing files:**

Reply:
sure